test/validation.js

```diff
@@ -57,6 +57,23 @@ describe('validation tests', function () {
         })
         .argv
     })
+
+    it('should ignore the --no- part if boolean negation is disabled', function (done) {
```

Comment:
If the test isn't async, you can just not accept `done` as an argument, and then drop the line:

```js
return done()
```